core/modules/node/node.types.inc

```diff
@@ -45,6 +45,11 @@ function node_overview_types() {
         'weight' => 10,
       );
     }
+    $links['node_layout'] = array(
```

Comment:
Do we need `if (user_access('administer layouts')) {` here?

Reply:
Yes. Based on checking `user_access('administer fields')` above, we probably need to check layout access here as well.